pennylane/ops/op_math/pow.py

```diff
@@ -107,24 +106,10 @@ class PowOperation(Operation):
     # until we add gradient support
     grad_method = None
 
-    def inv(self):
-        warn(
-            "In-place inversion with inv is deprecated. Please use qml.adjoint instead or qml.pow.",
-            UserWarning,
-        )
-        self.hyperparameters["z"] *= -1
-        self._name = f"{self.base.name}**{self.z}"
-        return self
-
     @property
     def inverse(self):
         return False
```

Comment:
same thing, base should always be False as well

Reply:
I'm going to have a second PR that removes the `inverse` property itself.  I broke it up into two parts to make this a "little" more manageable.